pkgs/development/libraries/gpgme/default.nix

```diff
@@ -31,22 +42,50 @@ stdenv.mkDerivation rec {
       url = "https://lists.gnupg.org/pipermail/gnupg-devel/attachments/20200415/f7be62d1/attachment.obj";
       sha256 = "00d4sxq63601lzdp2ha1i8fvybh7dzih4531jh8bx07fab3sw65g";
     })
+    # Support for Python 3.10, https://dev.gnupg.org/D545
+    (fetchpatch {
+      name = "support-python-310.patch";
+      url = "https://files.gnupg.net/file/data/vmjjll5eo6dfyqbyij4r/PHID-FILE-7ogk7dr565ogbttuhxdu/D545.diff";
```

Comment:
```suggestion
      url = "https://files.gnupg.net/file/data/kgt6y6sr2bro2xn25el5/PHID-FILE-hv7padcssgczan7xa6la/D545.diff";
```

Reply:
That looks like the old URL gets removed whenever new revision is pushed. We probably need to include the patch into the source tree.